src/manager/frames/safari/SafariBottom/SafariBottom.tsx

```diff
@@ -1,7 +1,21 @@
 import * as React from "react";
+import * as styles from "../style.scss";
+
+const Return = chrome.extension.getURL("assets/safariFrame/return.svg");
+const next = chrome.extension.getURL("assets/safariFrame/next.svg");
+const upload = chrome.extension.getURL("assets/safariFrame/upload.svg");
+const book = chrome.extension.getURL("assets/safariFrame/book.svg");
+const windows = chrome.extension.getURL("assets/safariFrame/windows.svg");
```

Comment:
what do you think about writing it cleaner like:
```ts
[
 Return, 
 next, 
 upload, 
 book, 
 windows
] = ['return' 'next' 'upload' 'book' 'windows']
 .map(_ => chrome.extension.getURL(`assets/safariFrame/${_}.svg`))
```

Reply:
I love it thanks! great idea